src/lipsticksettings.cpp

```diff
@@ -77,6 +78,21 @@ QSize LipstickSettings::screenSize()
     return QGuiApplication::primaryScreen()->size();
 }
 
+void LipstickSettings::exportScreenSize()
+{
+    const int defaultValue = 0;
+    MGConfItem widthConf("/lipstick/screen/primary/width");
+    if (widthConf.value(defaultValue) != QGuiApplication::primaryScreen()->size().width()) {
+        widthConf.set(QGuiApplication::primaryScreen()->size().width());
+        widthConf.sync();
```

Comment:
Is calling sync really required? I would have assumed the dtor took care of this